server/router/server.go

```diff
@@ -31,6 +31,8 @@ func NewRouter(_ context.Context, h models.HandlerInterface, port int, g http.Ha
 		Methods("GET")
 	gMux.Handle("/api/system/database", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.GetSystemDatabase), models.ProviderAuth))).
 		Methods("GET")
+	gMux.Handle("/api/system/database/reset", h.ProviderMiddleware(h.AuthMiddleware(h.SessionInjectorMiddleware(h.ResetSystemDatabase), models.ProviderAuth))).
+		Methods("GET")
```

Comment:
@Revolyssup we don't delete resources using a `GET`. Please offer better guidance here.

Reply:
@Revolyssup only engages about once a week, if that, on the project, so I'll point out the Guiding Principles section in the [Meshery API Endpoints](https://docs.google.com/spreadsheets/d/1ABJCbfQRi0uN_YoP2kmHZ-lTI4S4QuvRk1_unomKNRE/edit#) spreadsheet in the meantime.

Reply:
Thanks, @leecalcote.
We were using graphql and so `POST` request. And `GET` isn't appropriate.

Then should I use `DELETE` or `POST`?

Reply:
Thanks for this @Aritra8438. I think it could be debated, however, since there is only one database, let's go with `DELETE`.

Reply:
@leecalcote Yes I had also suggested DELETE in a previous resolved comment

Reply:
Oh, good, @Revolyssup. Weird, that the other two resolved comments don't include this. I'm sure there are other resolved comments that are missing in my view. Thanks for being on top of this. A GET with this sort of destructive power is an accident waiting to happen.

Reply:
> Yes I had also suggested DELETE in a previous resolved comment

I also missed this one, sorry for that.

Reply:
I don't see it either, though.